<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlotHistory</class>
 <widget class="QWidget" name="FormPlotHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotHistory</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,9">
     <item>
      <layout class="QVBoxLayout" name="vLay14">
       <item>
        <layout class="QHBoxLayout" name="hLay14">
         <item>
          <widget class="QLabel" name="label14">
           <property name="text">
            <string>curve 14bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev14">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext14">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLay14Status">
         <item>
          <widget class="QLabel" name="labelStatus14">
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLay14StatusGo">
           <item>
            <widget class="QLabel" name="label14Go">
             <property name="text">
              <string>go:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit14Go"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLay24">
       <item>
        <layout class="QHBoxLayout" name="hLay24">
         <item>
          <widget class="QLabel" name="label24">
           <property name="text">
            <string>curve 24bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev24">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext24">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLay24Status">
         <item>
          <widget class="QLabel" name="labelStatus24">
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLay24StatusGo">
           <item>
            <widget class="QLabel" name="label24Go">
             <property name="text">
              <string>go:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit24Go"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hLayOpt">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMix">
       <property name="text">
        <string>mix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSplit">
       <property name="text">
        <string>split</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
